e2e/e2etests/test_solana_withdraw_and_call.go

```diff
@@ -21,10 +21,16 @@ import (
 // in connected program pda and account provided in remaining accounts to demonstrate that lamports
 // can be moved to accounts in connected program as well as gateway program
 func TestSolanaWithdrawAndCall(r *runner.E2ERunner, args []string) {
-	require.Len(r, args, 1)
+	require.LessOrEqual(r, len(args), 3)
```

Comment:
do you want to make it more strict, e.g. `require.True(r, len(args) == 1 || len(args) == 3)`?